packages/sitecore-jss/src/site/graphql-error-handling-pages-service.ts

```diff
@@ -0,0 +1,100 @@
+import { GraphQLClient, GraphQLRequestClient } from '../graphql';
+import { siteNameError } from '../constants';
+import debug from '../debug';
+
+// The default query for request error handling
+const defaultQuery = /* GraphQL */ `
+  query ErrorHandlingPagesQuery($siteName: String!, $language: String!) {
+    site {
+      siteInfo(site: $siteName) {
+        errorHandling(language: $language) {
+          notFoundPagePath
+          serverErrorPagePath
+        }
+      }
+    }
+  }
+`;
+
+export type GraphQLErrorHandlingPagesServiceConfig = {
```

Comment:
Can we simplify the naming even more in these new services/types and get rid of "Handling"? i.e. `GraphQLErrorPagesServiceConfig` here